src/admin/templatetags/template_extras.py

```diff
@@ -0,0 +1,9 @@
+from django.template import Library, loader
+
+register = Library()
+
+
+@register.filter(name='center', is_safe=True, needs_autoescape=False)
+def center(value, arg=""):
+    template = loader.get_template("components/center.html")
+    return template.render({"value": value, "pad": arg})
```

Comment:
Maybe this can help?

https://stackoverflow.com/a/51986759

```suggestion
from django.template import Library, loader
from django.utils.safestring import mark_safe


register = Library()


@register.filter(name='center', is_safe=True, needs_autoescape=False)
def center(value, arg=""):
    template = loader.get_template("components/center.html")
    return mark_safe(template.render({"value": value, "pad": arg}))
```

Reply:
This doesn't work